### PR TITLE
Push develop to mastes

### PR DESCRIPTION
**Pull request by diraulo**
PT Story: no story

Changes proposed in this pull request:
- New app design

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -16,5 +16,8 @@
       .box
         %span.alert= alert
     = yield
-    .logo.col-center-block
-      = link_to image_tag('logo.png'), 'http://www.craftacademy.se', target: '_blank'
+    .logo.center
+      %p
+        A
+        = link_to 'CRAFT ACADEMY', 'http://www.craftacademy.se', target: '_blank'
+        PRODUCTION
```
Comment (tochman) [inline app/views/layouts/application.html.haml]: What is `PRODUCTION`?

Comment (diraulo) [inline app/views/layouts/application.html.haml]: okay @tochman this will read `A CRAFT ACADEMY PRODUCTION`

**Comment by diraulo** [comment]
If that was the only concern @tochman let's merge this in.. waiting on that to deploy :)

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (b264) [reply app/views/layouts/application.html.haml:23]: I think it's better to write in regular case in the semantic HTML and then do a `text-transform: uppercase;` with CSS

**Pull request merged**